charts/internal/shoot-storageclasses/templates/storageclasses.yaml

```diff
@@ -108,6 +108,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: CSI_LVM_PULL_POLICY
+          value: "ifnotpresent"
```

Comment:
Isnt this CamelCase ?

Reply:
```
const(
...
	pullIfNotPresent = "ifnotpresent"
```
and later
```
	if strings.ToLower(pullPolicy) == pullIfNotPresent {
		pp = v1.PullIfNotPresent
	}
```

Reply:
right, but k8s documentation says IfNotPresent, so should we.

Reply:
changed to IfNotPresent